src/com/oltpbenchmark/benchmarks/tpcc/procedures/NewOrder.java

```diff
@@ -152,22 +211,31 @@ public ResultSet run(Connection conn, Random gen,
     if (TPCCUtil.randomNumber(1, 100, gen) == 1)
       itemIDs[numItems - 1] = TPCCConfig.INVALID_ITEM_ID;
 
+    //initializing all prepared statements
```

Comment:
nit: space after `//`, start sentence with capital letter and `.` at the end.

Reply:
Done.